StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1505OpeningCurlyBracketsMustNotBeFollowedByBlankLine.cs

```diff
@@ -127,7 +127,7 @@ private void AnalyzeOpenBrace(SyntaxNodeAnalysisContext context, SyntaxToken ope
                 startIndex++;
             }
 
-            startIndex = startIndex % separatingTrivia.Count;
+            startIndex = (startIndex == separatingTrivia.Count) ? 0 : startIndex + 1;
```

Comment:
:memo: I tried to ask about this line after merging #962, but the diff went away (look at the files tab in that pull request) :confused: